Balance/Profile/ProfileView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ProfileView.swift
+//  Balance
+//
+//  Created by Gonzalo Perisset on 25/04/2023.
+//
+
+import Account
+import BalanceOnboardingFlow
+import SwiftUI
+
+struct ProfileView: View {
+    @State private var showingAvatarSheet = false
+    @EnvironmentObject var account: Account
+
+    var body: some View {
+        ZStack {
+            VStack(alignment: .center, spacing: 0) {
+                HeaderMenu(title: "")
+                    .background(primaryColor)
+                Button(action: {
+                    showingAvatarSheet.toggle()
+                    print("avatarView")
+                }) {
+                    profileView
+                }.sheet(isPresented: $showingAvatarSheet) {
+                    AvatarSelectionView()
+                }
+                
+                Spacer().frame(height: 50)
+                Text("Gonzalo Perisset")
```

Comment:
Gonza, I'm assuming this was for testing purposes? If so, can we use something else or perhaps leave it blank? Thanks!
@gperisset